### PR TITLE
Split and move API acceptance tests FLOC-2684

### PR DESCRIPTION
**Pull request by dwgebler**
Fixes FLOC-2684

### CODE REVIEW

File: flocker/acceptance/obsolete/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Acceptance tests for ``flocker`` components that are either deprecated
+or planned for deprection in the near future.
```
Comment (itamarst) [inline flocker/acceptance/obsolete/__init__.py]: typo: "deprection"

File: flocker/acceptance/obsolete/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
```
Comment (itamarst) [inline flocker/acceptance/obsolete/__init__.py]: ClusterHQ Inc.

**Comment by itamarst** [comment]
Looks good, merge when addressed and build is green.

File: flocker/acceptance/obsolete/__init__.py (hunk 2 above)
Comment (dwgebler) [inline flocker/acceptance/obsolete/__init__.py]: One day, in the distant future, I'll remember that we're not called Hybrid Logic Ltd anymore. Old habits.... :smile:

**Pull request merged**